### PR TITLE
Added new transform

### PR DESCRIPTION
**Pull request by bklare**
A subset of the [previous pull request](https://github.com/biometrics/openbr/compare/averagePoints) (sans eye drawing). This is a new pull request since the previous pull request is not valid after recent refactor.

### CODE REVIEW

**Comment by jklontz** [comment]
LGTM, your code quality has really improved :)

File: openbr/plugins/metadata/averagepoints.cpp [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+        for (int i = 0; i < indices.size(); i++) {
+            x1 += src.points()[indices[i]].x();
+            y1 += src.points()[indices[i]].y();
+        }
+
+        QPointF p(x1 / indices.size(), y1 / indices.size());
+        if (!metaName.isEmpty())
+            dst.set(metaName, p);
+        if (append)
+            dst.points().append(p);
```
Comment (sklum) [inline openbr/plugins/metadata/averagepoints.cpp]: As a heads up @bklare, `br::File` has the convenience methods `appendPoint(QPointF)` and `appendPoints(QPointF)`.

Comment (jklontz) [inline openbr/plugins/metadata/averagepoints.cpp]: Good observation, in fact this code doesn't work as written, `points()` returns a copy, not a reference.

**Comment by bklare** [comment]
@sklum  @jklontz Thanks for the catch. Clearly this was an option I wasn't using. Fixed. Will merge now.

**Pull request merged**